edx/edx-enterprise
# Increase font size on DSC page.

**Description:** The default font size at edX is 16px for readability, but DSC page was using 14px. This patch updates DSC font size to 16px.

**JIRA:** [ENT-764](https://openedx.atlassian.net/browse/ENT-764)

**Testing instructions:**

1. Log in as an enterprise user.
2. Go to a course which requires data sharing consent to view the DSC page.
3. Verify that the paragraph font size is 16px.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**

**Author notes and concerns**:

There's a [CSS rule which sets default font-size to 14px on all enterprise pages](https://github.com/edx/edx-enterprise/blob/f231808af6cb2c177989701fc78382348e3c2a6b/enterprise/static/enterprise/sass/_base.scss#L84-L86). I tried removing it, but that causes some layout issues on course and program landing pages, so I decided to override the default font size for the DSC page instead since ENT-764 is explicitly about the DSC page.

If desired, I could remove that global enterprise default and try to fix layout issues on other enterprise pages instead.
